### PR TITLE
Better content extraction of title and description for GitHub URLs

### PR DESCRIPTION
**Pull request by jacksonh**

### CODE REVIEW

File: packages/content-handler/src/websites/github-handler.ts [hunk 1]
```diff
@@ -12,10 +12,29 @@ export class GitHubHandler extends ContentHandler {
 
   async preParse(url: string, dom: Document): Promise<Document> {
     const body = dom.querySelector('body')
+    const head = dom.querySelector('head')
```
Comment (sywhb) [inline packages/content-handler/src/websites/github-handler.ts:15]: Where is `head` being used?

File: packages/content-handler/src/websites/github-handler.ts [hunk 2]
```diff
@@ -12,10 +12,29 @@ export class GitHubHandler extends ContentHandler {
...
+      body.replaceChildren(article)
+
+      // Attempt to set the author also. This is available on repo homepages
+      // but not on things like PRs. Ideally we want PRs and issues to have
+      // author set to the author of the PR/issue.
+      if (linkAuthor && linkAuthor.textContent) {
+        const author = dom.createElement('span')
+        author.setAttribute('rel', 'author')
+        author.innerHTML = linkAuthor.textContent
+        article.appendChild(author)
```
Comment (sywhb) [inline packages/content-handler/src/websites/github-handler.ts:30]: Nice!

File: packages/content-handler/src/websites/github-handler.ts [hunk 3]
```diff
@@ -12,10 +12,29 @@ export class GitHubHandler extends ContentHandler {
...
+      }
+    }
+
+    // Remove the GitHub - and repo org from the title
+    const twitterTitleContent = twitterTitle?.getAttribute('content')
+    if (twitterTitle && twitterTitleContent) {
+      twitterTitle.setAttribute('content', twitterTitleContent.replace(/GitHub - .*\//, ''))
```
Comment (sywhb) [inline packages/content-handler/src/websites/github-handler.ts:37]: The `content` is like "omnivore: Omnivore is a complete, open source read-it-later solution for people who like reading.". Do we want to keep the repo name?

**Review by sywhb — APPROVED** [review]
LGTM 👍

File: packages/content-handler/src/websites/github-handler.ts (hunk 3 above)
Comment (jacksonh) [reply packages/content-handler/src/websites/github-handler.ts:37]: @sywhb the idea is we will get the repo name as the author.

File: packages/content-handler/src/websites/github-handler.ts (hunk 2 above)
Comment (jacksonh) [reply packages/content-handler/src/websites/github-handler.ts:30]: Took so long to find 😬

**Pull request merged**